app/code/Magento/InventoryInStorePickupShipping/composer.json

```diff
@@ -7,6 +7,7 @@
         "magento/module-inventory-in-store-pickup-shipping-api": "*",
         "magento/module-inventory-in-store-pickup-api": "*",
         "magento/module-inventory-sales-api": "*",
+        "magento/module-inventory-in-store-pickup-quote": "*",
```

Comment:
Could we somehow remove this dependency?

I prefer to avoid such dependencies:
`magento/module-inventory-in-store-pickup-quote` -> `magento/module-inventory-in-store-pickup-shipping-api`
`magento/module-inventory-in-store-pickup-shipping` -> `magento/module-inventory-in-store-pickup-shipping-api`
`magento/module-inventory-in-store-pickup-shipping` -> `magento/module-inventory-in-store-pickup-quote`

Reply:
ok